### ISSUE TITLE
Clickable Box with focusIndicator=false still has focus outline when clicked

### ISSUE THREAD

**Issue by johnliu55tw** [grommet/grommet#4543]
### Expected Behavior
`Box` with `focusIndicator=false` should not contain any focus outline after clicked.

### Actual Behavior
`Box` with `focusIndicator=false` still has a black focus outline. Seems like it falls back to the user-agent stylesheet.

### URL, screen shot, or Codepen exhibiting the issue
![image](https://user-images.githubusercontent.com/5047724/93963446-c5dd8080-fd8f-11ea-9f6f-4b76492135f4.png)
![image](https://user-images.githubusercontent.com/5047724/93963469-d1c94280-fd8f-11ea-87db-0bb3fa2c91b1.png)
https://codesandbox.io/s/grommet-v2-template-clickable-box-3oit6?file=/index.js

### Steps to Reproduce
1. Create a `Box` with props `onClick={ () => {} }` and `focusIndicator={ false }`\
2. Click the `Box`
3. A black outline shown

### Your Environment
- Grommet version: 2.15.0
- Browser Name and version: Chromium 85.0.4183.102
- Operating System and version (desktop or mobile): ArchLinux

**Comment by IanKBovard** [comment]
The user-agent stylesheet is a browsers default style rules. focusIndicator false is working as intended, it removes the green border that is applied when focusing an element. Here is a codesandbox showing focusindicator, and one fix to remove the focus that chrome applies by default when focusindicator is set to false.

https://codesandbox.io/s/grommet-v2-template-clickable-box-forked-2rquy?file=/index.js

**Comment by johnliu55tw** [comment]
Thank you for the quick solution!

But I'm thinking maybe the `focusIndicator=false` can help also remove the default outline? That would be very helpful!

**Comment by IanKBovard** [comment]
I don't think that's a good idea. Overriding browser defaults should be up to the user.

**Comment by johnliu55tw** [comment]
Sounds reasonable. Thanks for the solution!
